tensorflow/core/kernels/mkl/mkl_eltwise_activation_base_op.h

```diff
@@ -0,0 +1,288 @@
+/* Copyright 2021 The TensorFlow Authors. All Rights Reserved.
```

Comment:
Please add a newline after this line. Same for other new files.

Reply:
Done